tests/test_x86.py

```diff
@@ -598,3 +598,192 @@ def run_and_check(x, result):
         ),
         np.array([0.11827405109974021], dtype=np.float32),
     )
+
+
+def test_mm256_broadcast_ss_scalar(compiler):
+    @proc
+    def mm256_broadcast_ss_scalar_wrapper(out: f32[8] @ DRAM, val: f32[1] @ DRAM):
+        tmp_val: f32
+        tmp_val = val[0]
+        tmp_buffer_0: f32[8] @ AVX2
+        mm256_loadu_ps(tmp_buffer_0, out)
+        mm256_broadcast_ss_scalar(tmp_buffer_0, tmp_val)
+        mm256_storeu_ps(out, tmp_buffer_0)
+        val[0] = tmp_val
+
+    @proc
+    def mm256_broadcast_ss_scalar_ref(out: f32[8] @ DRAM, val: f32[1] @ DRAM):
+        # @instr {out_data} = _mm256_broadcast_ss(*{val});
+        assert stride(out, 0) == 1
+        for i in seq(0, 8):
+            out[i] = val[0]
+
+    fn = compiler.compile(
+        [mm256_broadcast_ss_scalar_wrapper, mm256_broadcast_ss_scalar_ref],
+        skip_on_fail=True,
+        CMAKE_C_FLAGS="-march=skylake",
+    )
+
+    out = np.array(
+        [
+            0.8352003,
+            0.119042955,
+            0.05500923,
+            0.9115426,
+            0.8030574,
+            0.07465152,
+            0.5667018,
+            0.53294945,
+        ],
+        dtype=np.float32,
+    )
+    val = np.array([2.44])
+    out_copy = out.copy()
+    val_copy = val.copy()
+
+    getattr(fn, "mm256_broadcast_ss_scalar_wrapper")(None, out, val)
+    getattr(fn, "mm256_broadcast_ss_scalar_ref")(None, out_copy, val_copy)
+
+    np.testing.assert_almost_equal(out, out_copy)
+    np.testing.assert_almost_equal(val, val_copy)
+
+
+def test_mm256_add_ps(compiler):
```

Comment:
These tests need to be marked with `@pytest.mark.isa("AVX2")`

Reply:
See here for an example:

https://github.com/exo-lang/exo/blob/c56d978350bc5a524d2bd932fd9ae4e313a0f1ce/tests/test_x86.py#L23-L40

Reply:
Done